### PR TITLE
Create bug bounty program page in Lisk docs - Closes #1506

### LINKED ISSUE #1506: Add page about the bug bounty program
### Is your feature request related to a problem?
The bug bounty page from lisk.com will be removed, so we need to add it to the Lisk docs.

### Description
A page explaining the Lisk bounty program, analog to the currently existing page https://lisk.com/bug-bounty-program

### PR DESCRIPTION
**Pull request by chris529**
Closes #1506

### CODE REVIEW

File: docs/modules/ROOT/pages/intro/bug-bounty-program.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+All developers and researchers who report security vulnerabilities and bugs in Lisk Core may be eligible for a payment in accordance with our guidelines which are explained in more detail on this page.
+Firstly, please ensure before making any submissions that you have reviewed our bug bounty program policy rules below.
+
+== Guidelines
+
+=== Focus on Lisk Core
+
+Only vulnerabilities and bugs in Lisk Core are being considered.
+Focus on the master branch and, if one exists, the latest Betanet branch only.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/intro/bug-bounty-program.adoc]:
```suggestion
Focus on the`main` branch and, if one exists, the latest Betanet branch only.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looking good, but the new page still needs to be added to the `nav.adoc`.

Because there is no intro section in v5 docs, please add the new page right above the Glossary in the navigation, thanks!

File: docs/modules/ROOT/pages/bug-bounty-program.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Chris Braithwaite <christopher.braithwaite@lightcurve.io> Mona Bärenfänger <mona@lightcurve.io>
+:description: This page covers the submission guidelines regarding how to receive remuneration for locating and reporting security vulnerabilities and bugs in the Lisk core.
+:toc: preamble
+:idprefix:
+:idseparator: -
+:imagesdir: ../../assets/images
+
+:url_submission_form: https://lisk.com/contact/ive-found-bug/form
+
+image::intro/bugbounty.png[align="center"]
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/bug-bounty-program.adoc]: Overall it looks good 👍🏼. However, the quality of the images used is compromised. Please use high-definition images (png/SVG).

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/bug-bounty-program.adoc (hunk 2 above)
Comment (Tschakki) [reply docs/modules/ROOT/pages/bug-bounty-program.adoc]: agree! Also, PNG with transparent background might look better here

File: docs/modules/ROOT/pages/bug-bounty-program.adoc [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+All developers and researchers who report security vulnerabilities and bugs in Lisk Core may be eligible for a payment in accordance with our guidelines which are explained in more detail on this page.
+Firstly, please ensure before making any submissions that you have reviewed our bug bounty program policy rules below.
+
+== Guidelines
+
+=== Focus on Lisk Core
+
+Only vulnerabilities and bugs in Lisk Core are being considered.
+Focus on the`main` branch and, if one exists, the latest Betanet branch only.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/bug-bounty-program.adoc]:
```suggestion
Focus on the `main` branch and, if one exists, the latest Betanet branch only.
```

**Review by Tschakki — APPROVED** [review]

File: docs/modules/ROOT/pages/bug-bounty-program.adoc (hunk 2 above)
Comment (chris529) [reply docs/modules/ROOT/pages/bug-bounty-program.adoc]: OK, I have now recreated the diagrams, they are all now print quality .png files and all have a transparent background 👍

**Review by TalhaMaliktz — APPROVED** [review]
Thanks :). LGTM 👍🏼

**Pull request merged**